specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspProfileGet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Get NSP Profile",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1",
+    "profileName": "profile1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "nsp1",
+        "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityPerimeters/nsp1/profiles/profile1",
+        "type": "Microsoft.Network/networkSecurityPerimeters/profiles",
+        "location": "resource location",
```

Comment:
This should be "perimeter location". Also need to check whether we will be removing this or not.